src/ProductView/FilterBar/FilterBar.js

```diff
@@ -1,14 +1,14 @@
 import React, { Component } from 'react';
-import Filter from './Filter/Filter.js';
+import InputField from '../../Utils/InputField.js';
 
 class FilterBar extends Component {
     render() {
         return (
             <div className="filter-bar container">
                 <div className="form-group row">
                     {this.props.filter.map((filter, index) =>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "index". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)